src/main/webapp/style.css

```diff
@@ -1,38 +1,39 @@
-* {
+/* Global styling */
+
+@font-face {
   font-family: 'Product Sans';
+  font-style: normal;
+  font-weight: 400;
+  src: url('./fonts/product-sans/ProductSansRegular.ttf') format('truetype');
 }
 
-body {
-  margin-bottom: 60px !important;
+* {
+  font-family: 'Product Sans';
 }
 
+body,
```

Comment:
I believe these should be alphabetized. 

https://google.github.io/styleguide/htmlcssguide.html#Declaration_Order

Reply:
I think that only applies to the declarations within the brackets--the order of selectors matters somewhat when overriding prior ones.  But this is an easy change and consistent with everything else so I'll switch em :)